scanpipe/api/views.py

```diff
@@ -497,6 +497,22 @@ def license_clarity_compliance(self, request, *args, **kwargs):
         clarity_alert = project.get_license_clarity_compliance_alert()
         return Response({"license_clarity_compliance_alert": clarity_alert})
 
+    @action(detail=True, methods=["get"])
+    def scorecard_compliance(self, request, *args, **kwargs):
```

Comment:
This needs a new entry in the REST API documentation, see `_rest_api_license_clarity_compliance`